doc/site/content/en/docs/Architecture/Components/Connectors/file_connector.md

```diff
@@ -18,22 +18,43 @@ specify the maximum number of files (`maxNumOfPages`) that Lucille will load int
-you want to use will be a map within the `fileOptions` you specify. You can use `csv`, `json`, `xml`, and `parquet` (coming soon!). 
+you want to use will be a map within the `fileOptions` you specify. You can use `csv`, `json`, or `xml`.
 See the documentation for each File Handler to see what arguments are needed / accepted.
 
-### Parameters for Traversal
-There are many different parameters you can specify to customize your traversal of the given path. A few notable ones:
-
-* `includes`/`excludes`: A list of regex patterns for files you want to include in / exclude from your traversal.
-* `fileOptions.getFileContent`: Whether, during traversal, the FileConnector should add an array of bytes representing the file's contents to the Lucille document it publishes.
-**This will slow down traversal significantly and is resource intensive. On the cloud, this _will_ download the file contents.**
-* `fileOptions.moveToAfterProcessing`: A path to move files to after processing. **Only works for the local file system.**
-* `fileOptions.moveToErrorFolder`: A path to move files to if an error occurs. **Only works for the local file system.**
\ No newline at end of file
+### File Options
+File options determine **how** you handle and process files you encounter during a traversal. Some commonly used options include:
+* `getFileContent`:  Whether, during traversal, the FileConnector should add an array of bytes representing the file's contents to the Lucille document it publishes.
+                     **This will slow down traversal significantly and is resource intensive. On the cloud, this _will_ download the file contents.**
+* `handleArchivedFiles`/`handleCompressedFiles`: Whether you want to handle archive or compressed files, respectively, during your traversal. For cloud files, this _will_ download the file's contents.
+* File Handlers: You can include `csv`, `json`, or `xml`, enabling additional processing for these file types. Their configuration will be used to build/configure the File Handler.
+
+### Filter Options
+Filter options determine **which** files will/won't be processed & published in your traversal. All filter options are optional. 
+If you specify multiple filter options, files must comply with all of them to be processed & published.
+* `includes`: A list of patterns for the only file names that you want to include in your traversal.
+* `excludes`: A list of patterns for file names that you want to exclude from your traversal.
+* `lastModifiedCutoff`: Filter out files that haven't been modified recently. For example, specify `"1h"`, and only
+files modified within the last hour will be processed & published.
+* `lastPublishedCutoff`: Filter out files that were recently published by Lucille. For example, specify `"1h"`, and only
+files published by Lucille more than an hour ago will processed & published. Requires you to provide **state** configuration, otherwise,
```

Comment:
...only files published by Lucille more than an hour ago _(or never published)_ will be processed...